common/crypto/openssl/cert.c

```diff
@@ -1019,3 +1033,333 @@ oe_result_t oe_cert_get_validity_dates(
 
     return result;
 }
+
+#ifdef OE_BUILD_ENCLAVE
+/*
+ * Parse the name string into X509_NAME struct. The format of the string is
```

Comment:
It's bad form to put OE_BUILD_ENCLAVE-only functions in the common directory.
This should move to a file in the enclave directory outside of common.
(Ideally, one would never see OE_BUILD_ENCLAVE used anywhere in the common directory.)